person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] Let's change the `:id` attribute to the read-only attributes. This is because id's are usually autogenerated and unique, and we want to avoid replacing it to avoid bugs.